timemachine/fe/free_energy.py

```diff
@@ -929,6 +929,9 @@ def log_q_fn(replica_idx: ReplicaIdx, state_idx: StateIdx) -> float:
     def get_equilibrated_xvb(xvb: CoordsVelBox, params: NDArray) -> CoordsVelBox:
         if md_params.n_eq_steps == 0:
             return xvb
+        # Set the movers to 0 to ensure they all equilibrate the same way
```

Comment:
suggestion for clarity:
```suggestion
        # Ensure initial mover state is consistent across replicas
```